recipe/meta.yaml

```diff
@@ -1,82 +1,77 @@
 {% set name = "category_encoders" %}
-{% set version = "2.2.2" %}
+{% set version = "2.5.0" %}
+{% set sha256 = "ff72d21ddf62d829f43375c67ea4934c82947f9a0c5cbf9114838624db941420" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 79651dc3e8cd6749bc5d404c49a79e93469016d106736326842305e076136cb0
-    folder: dist
   - url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: b8fa5f2c2f3294ed7847e9c5486566604a6a321308dfb5377440a7ca2a9a5b1f
-    folder: src
+    sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
   number: 0
+  skip: true  # [py<37 or py>310]
   script:
-    - cd dist && {{ PYTHON }} -m pip install -vvv .
+    - {{ PYTHON }} -m pip install --no-deps -vvv .
 
 requirements:
   host:
-    - numpy >=1.14.0
-    - pandas >=0.21.1
-    - patsy >=0.5.1
+    - python
     - pip
-    - python >=3.5
+    - wheel
+    - setuptools
+    - numpy >=1.14
     - scikit-learn >=0.20.0
     - scipy >=1.0.0
     - statsmodels >=0.9.0
-  run:
-    - numpy >=1.14.0
-    - pandas >=0.21.1
+    - pandas >=1.0.5
     - patsy >=0.5.1
-    - python >=3.5
+    - unittest2
+  run:
+    - python
+    - numpy >=1.14
     - scikit-learn >=0.20.0
     - scipy >=1.0.0
     - statsmodels >=0.9.0
+    - pandas >=1.0.5
+    - patsy >=0.5.1
+    - unittest2
 
 test:
   source_files:
-    - src/tests
+    - tests
   requires:
-    - cython
     - pip
     - pytest
-    - pytest-cov
   imports:
```

Comment:
cython is no longer used in category_encoders in testing or building. pytest-cov is not necessary if it is not being acted on.

Reply:
ok